cea/demand/set_point_from_predefined_file.py

```diff
@@ -29,7 +29,8 @@ def calc_set_point_from_predefined_file(tsd, bpr, weekday, building_name, locato
 
     The excel files with the predefined hourly temperatures are to be stored in
     `scenario\inputs\predefined-hourly-setpoints` folder. This folder in turn has two subfolders namely `space heating`
-    and `space cooling`. Inside these folders the setpoints are to be provided in an excel file.
+    and `space cooling`. Inside these folders the setpoints are to be provided in an excel file. The excel files need
```

Comment:
can we keep whitespace out of folder names please? `space-cooling` and `space-heeting` would work just as well and potentially create less problems for future work.